charts/flyte-core/values-eks.yaml

```diff
@@ -249,6 +250,15 @@ configmap:
         storage: 2000Mi
 
   core:
+    # -- follows the structure specified [here](https://pkg.go.dev/github.com/flyteorg/flytepropeller/manager/config#Config).
+    manager:
+      pod-application: "flytepropeller"
+      pod-template-container-name: "flytepropeller"
+      pod-template-name: "flytepropeller-template"
+      scan-interval: 10s
+      shard:
+        type: Hash
+        pod-count: 3
```

Comment:
Looks like these are the defaults here: https://github.com/flyteorg/flytepropeller/blob/v0.16.7/manager/config/config.go#L13-L25.. unless you need to reference them in the deployment, you don't need to explicitly set them here... makes it a bit easier to change the defaults if needed...

Reply:
I removed the "scan-interval" and "shard" configuration options, but kept the others. I think it makes sense to them in places within the PodTemplate (e.x., [here](https://github.com/flyteorg/flyte/blob/0ce73769842d8046174507daf726943c886c0c15/charts/flyte-core/templates/propeller/manager.yaml#L37) so that the linking between the manager and the PodTemplate is automatic and users don't have to worry about mis-configuration. If you think this might be confusing we can certainly change it?

Reply:
Makes sense... anything you are referencing in the chart you can keep... that linkage makes sense! thank you